modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart

```diff
@@ -12,12 +12,16 @@ class Emails {
       Serverpod.instance.getPassword('email_password_salt') ??
       'serverpod password salt';
 
+  static String? get _pepper =>
+      Serverpod.instance.getPassword('emailPasswordPepper');
```

Comment:
Suggestion: I think it would be nice to centralize these strings, makes it easier to get an overview of what we have implemented / support. Makes it more obvious if we forgot to document something.